codespell_lib/_codespell.py

```diff
@@ -40,6 +40,7 @@
 quiet_level = 0
 encodings = ['utf-8', 'iso-8859-1']
 word_regex = re.compile(r"[\w\-']+")
+c_escape_regex = re.compile(r'\\\w')
```

Comment:
Would it be better to use the [standard set of escape sequences](https://en.wikipedia.org/wiki/Escape_sequences_in_C#Table_of_escape_sequences)?